### DISCUSSION TITLE
article/fumy5wdd/

### DISCUSSION THREAD

**Discussion by giscus** [yxzlwz/blog#3] [Announcements]
# article/fumy5wdd/

先说需求：我在家里有一台服务器，但是没有静态IP。之前使用的是花生壳免费的服务，但是后来收费了，就想到自己实现一个。 参考资料： API获取自己的公网IP地址 API查询IP信息异想之旅的博客-CSDN博客获取公网ip的api (https://blog.csdn.net/weixin_44495599/article/details/11603549...

**Comment by yellowbutton** [comment]
前四行记得改，记得给ram用户权限，记得设置主机名称，包安装不上百度，以上

**Reply by yxzlwz** [reply]
依赖项如下：

```
aliyun-python-sdk-alidns
aliyun-python-sdk-core
aliyun-python-sdk-core-v3
```
